src/container/plugins/PausePlugin.js

```diff
@@ -45,7 +53,7 @@
 			{
 				this._paused = paused;
 
-				if (this.client)
+				if ((this.client) && (!!this._disablePaused))
```

Comment:
this if statement needs to be wrapping all the contents of the set function for the disable of pause to work properly... as if it is not it will still change the pause button state and still show the pause overlay